monkey/infection_monkey/utils/linux/shell_startup_files_modification.py

```diff
@@ -0,0 +1,13 @@
+BASH_STARTUP_FILES = ["~/.bashrc", "~/.profile", "~/.bash_profile"]
```

Comment:
Contents of this file (`monkey/infection_monkey/utils/linux/shell_startup_files_modification.py`) are only relevant to `shell_startup_files` PBA. That means we shouldn't have this file under `monkey/infection_monkey/utils`. Create a directory/module: `monkey/infection_monkey/post_breach/shell_startup_files` and move all your relevant code there, except ` monkey/infection_monkey/post_breach/actions/modify_shell_startup_files.py`